src/app/app.component.html

```diff
@@ -4,6 +4,7 @@ <h1>Ngx-Stories Demo</h1>
 <ngx-stories
 [options]="{ width: 338, height: 600 }"
 [storyGroups]="storyGroups"
+[options]="ngxOptions"
```

Comment:
We're already passing options just above, can you please update it, so that we can use a single options property ?

Reply:
Updated the PR with required changes.